polaris/hub/client.py

```diff
@@ -178,41 +195,10 @@ def _normalize_owner(
     #     Overrides
     # =========================
```

Comment:
**Documentation**: This "header" isn't correct anymore, since some of the overrides are placed above it. Feel free to remove it too. Don't think it's needed, especially now that this PR simplifies the code by splitting it over different classes!